src/ur3e_hande_moveit_config/config/kinematics.yaml

```
@@ -0,0 +1,8 @@
+/**:
+  ros__parameters:
+    robot_description_kinematics:
+      ur_arm:
+        kinematics_solver: kdl_kinematics_plugin/KDLKinematicsPlugin
+        kinematics_solver_search_resolution: 0.0050000000000000001
+        kinematics_solver_timeout: 0.0050000000000000001
```

Drop the unnecessary precision. 
```suggestion
        kinematics_solver_search_resolution: 0.005
        kinematics_solver_timeout: 0.005
```

---

Changed:
https://github.com/maffettone/erobs/blob/713573eeb9b4f04c3dcd55634bd55a72deedef19/src/ur3e_hande_moveit_config/config/kinematics.yaml#L6-L7